trim.production.js

```diff
@@ -31,16 +31,24 @@ function createRecord (title, container, extension, fileData, alternativeContain
     throw new Error('alternativeContainers must be a list of strings: ' + alternativeContainers)
   }
 
+  var body = {
+    Title: title,
+    Container: container,
+    RecordExtension: extension,
+    AlternativeContainers: alternativeContainers,
+    Record: fileData
+  }
+
+  body = R.evolve({
+    Title: R.ifElse(R.isNilOrEmpty, R.always('Untitled Document'), R.replace(/:/g, '-')),
+    RecordExtension: R.replace(/./g, ''),
```

Comment:
The `.` in the original regex was matching any char, and then it was global, so it was pretty much equivalent to `/.*/` (at least the result was the same)